mpas_analysis/default.cfg

```diff
@@ -2577,6 +2578,132 @@ compareAsContoursOnSinglePlot = True
 contourLevelsResult = [1027.2, 1027.4, 1027.6, 1027.7, 1027.8, 1027.85, 1027.9, 1027.95, 1028.0]
 
 
+[osnapTransects]
+## options related to plotting model vs. World Ocean Circulation Experiment
+## (WOCE) transects.
+
+# Times for comparison times (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['ANN']
+
+# The approximate horizontal resolution (in km) of each transect.  Latitude/
+# longitude between observation points will be subsampled at this interval.
+# Use 'obs' to indicate no subsampling. Use 'mpas' to indicate plotting of
+# model data on the native grid, in which case comparison with observations
+# will take place on the observation grid.
+#horizontalResolution = mpas
+#horizontalResolution = obs
+horizontalResolution = 5
+
+# Horizontal bounds of the plot (in km), or an empty list for automatic bounds
+# The bounds are a 2-element list of the minimum and maximum distance along the
+# transect.  Note: A21=Drake Passage; A23=South Atlantic; A12=Prime Meridian
+horizontalBounds = {'WOCE_A21': [],
+                    'WOCE_A23': [],
+                    'WOCE_A12': []}
```

Comment:
I think you can just get rid of this.  The horizontal bounds can be the whole transect unless you think you would ever want to plot part of it.